lean-client-js-browser/src/inprocess.ts

```diff
@@ -109,7 +109,7 @@ export class InProcessTransport implements Transport {
             const BrowserFS = new FS();
             BrowserFS.initialize(libraryFS);
             const BFS = new EmscriptenFS(Module.FS, Module.PATH, Module.ERRNO_CODES, BrowserFS);
-            Module.FS.createFolder(Module.FS.root, 'library', true, true);
+            (Module.FS.createPath || Module.FS.createFolder)(Module.FS.root, 'library', true, true);
```

Comment:
The alternative here would be to inline the source code of `createFolder`, which is a `PATH.join` and a `mkdir`.

We don't have `PATH` in the current Lean build, but hopefully will have it in 3.51.1

Reply:
Ah, nevermind; that means we can't use `PATH` because it wouldn't be backwards-compatible.